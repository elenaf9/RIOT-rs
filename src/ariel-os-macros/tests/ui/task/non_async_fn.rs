#![no_main]
#![feature(impl_trait_in_assoc_type)]
#![feature(used_with_arg)]

// FAIL: the function must be async
#[ariel_os::task]
fn main() {}
