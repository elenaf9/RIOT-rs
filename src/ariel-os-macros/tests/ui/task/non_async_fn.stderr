error: custom attribute panicked
 --> tests/ui/task/non_async_fn.rs:6:1
  |
6 | #[ariel_os::task]
  | ^^^^^^^^^^^^^^^^^
  |
  = help: message: the function must be async
